woocommerce-payments.php

```diff
@@ -53,3 +56,32 @@ function wcpay_init() {
 
 // Make sure this is run *after* WooCommerce has a chance to initialize its packages (wc-admin, etc). That is run with priority 10.
 add_action( 'plugins_loaded', 'wcpay_init', 11 );
```

Comment:
Please add a comment saying that if they change the priority here they need to do in wcpay_check_old_jetpack_version too

Reply:
Added in https://github.com/Automattic/woocommerce-payments/pull/760/commits/8aa0aa5129af79cc137601d241c5cd28c9d0cdfb